content/docs/concepts/accountability/_index.md

```diff
@@ -0,0 +1,381 @@
+
+There are protocol constraints on when an _accusation_ can be made. A _reporting_ validator can only submit an _accusation_ _proof_ if:
+
+- the _offending_ validator:
+  - does not already have a _fault_ in the epoch in which the _new accusation_ is being made for an offence with a higher _severity_
+  - is not currently already accused of committing a rule infraction, i.e. there is a _pending accusation_. In this case, a _new accusation_ cannot be made until expiry of the _innocence proof window_ during which an accused _offending_ validator is able to submit an `innocence` proof refuting the _pending accusation_. This creates a _deadline_ measured in block height before which a new `accusation` proof cannot be submitted.
+- it is within the _accusation window_:
+  - an accusation must be made `<= 256` blocks after the detected accountability event.
+
+As each block is finalised, AFD will attempt to promote _accusations_ where the _innocence proof submission window_ has expired within protocol constraints to proven _faults_.
+
+To check if an _offending_ validator has a _pending accusation_, a _reporting_ validator calls protocol functions:
+
+- [`canAccuse()`](/reference/api/accountability/#canaccuse): to determine if (a) an offending validator is accusable, and, (b) the _deadline_ number of blocks remaining in the _innocence proof submission window_ if there is a _pending accusation_.
+- [`canSlash()`](/reference/api/accountability/#canslash): to determine if the  _offending_ validator already has a _fault_ for a rule infraction in the epoch with a _severity_ higher than the new accusable rule infraction detected.
+
+After successful [handling and verification](/reference/api/aut/op-prot/#handleevent-accountability-contract) of an _accusation_ on-chain a `NewAccusation` event is emitted logging the _offending_ [validator identifier](/concepts/validator/#validator-identifier) address, _severity_ of rule infraction, and the event ID.
+
+#### Innocence
+
+An _innocence_ is a claim refuting an _accusation_ for an accusable rule infraction. An _innocence_ is made by an _offending_ validator to prove innocence from a _pending accusation_ against it. Claims of _innocence_ are made by submitting _proof_ of innocence of an accusable rule infraction on-chain as an event of type `Innocence`. If successful, a proof of innocence cancels the _pending accusation_.
+
+The _offending_ validator has a limited time window to submit a _proof_ of _innocence_, otherwise, the _accusation_ may be _promoted_ to a _fault_ and be slashable end of epoch.
+
+An _innocence_ must be submitted within an _innocence proof submission window_ to be accepted, a designated number of blocks set as a protocol parameter. The window begins at the block height number at which the _accusation_ states the rule infraction occurred.
```

Comment:
```suggestion
An _innocence_ must be submitted within an _innocence proof submission window_ to be accepted, a designated number of blocks set as a protocol parameter. The window begins at the block height number at which the _accusation_ was reported.
```

The innocence window start at reportingBlock

Reply:
Good catch.